drivers/adc/ad400x/ad400x.h

```diff
@@ -40,7 +40,13 @@
 #ifndef SRC_AD400X_H_
 #define SRC_AD400X_H_
 
+#include <stdbool.h>
```

Comment:
is this include related to the commit description/subject?

Reply:
Yes, It is required.
* In ad400x.h, the ad400x_init_param struct contains few members of type bool.
* When spi_engine.h is included in ad400x.h, it internally includes spi_engine_private.h and it contains <stdbool.h>
* But, When we add compile time switch USE_STANDARD_SPI, it is required to include <stdbool.h> in header file ad400x.h.